client/src/mdn.scss

```
@@ -77,4 +77,614 @@ div.content {
   .live-sample-frame {
     max-width: 42rem;
   }
+
+// BCD TABLE
+  .bc-table {
+      width: 100%;
+      table-layout: fixed;
+      border: none;
+      border-collapse: separate;
+      border-spacing: 0;
+      background-color: #eaeef2;
+      background-color: rgba(212,221,228,.5);
+  }
+
+  .bc-table {
+    margin-bottom: 0;
+  }
+
+  .bc-table th {
+    background-color: transparent;
+    overflow: hidden;
+    padding: 5px 3px 5px 6px;
+  }
+
+  .bc-table tr {
+    position: relative;
+    border-bottom: 0 solid transparent;
+  }
+
+  .bc-table thead td {
+    border-top: 0;
+    border-right: 2px solid #222;
+    border-left: 0;
+  }
+
+  .bc-table tbody th, .bc-table thead td {
+    overflow: hidden;
+  }
+
+  .bc-table td, .bc-table th {
+    position: relative;
+    padding: 5px 3px;
+    border-color: #222;
+    border-top: 1px solid #222;
+    border-right: 0;
+    border-bottom: 0 solid transparent;
+    border-left: 1px solid #222;
+    text-align: center;
+  }
+
+  .bc-table tbody th {
+    border-right: 2px solid #222;
+    border-left: 0;
+    text-align: left;
+  }
+
+  .bc-table, .bc-table tbody, .bc-table td, .bc-table th, .bc-table thead, .bc-table tr {
+    box-sizing: border-box;
+  }
+
+  .bc-table tbody th+td, .bc-table thead td+th {
+    border-left: 0;
+  }
+
+  .bc-table-web td:nth-child(8), .bc-table-web th:nth-child(8) {
+    border-left: 2px solid #222;
+  }
+
+  .bc-table tbody th+td, .bc-table thead td+th {
+    border-left: 0;
+  }
+
+  .bc-table-web tbody td, .bc-table-web thead th {
+    width: 7.69231%;
+  }
+
+  .bc-table-web tbody th, .bc-table-web thead td {
+    width: 200px;
+  }
+
+  .bc-table tbody td[tabindex], .no-js .bc-table tbody td {
+    padding-top: 6px;
+    padding-bottom: 12px;
+  }
+
+  .bc-table td[tabindex] {
+    cursor: pointer;
+  }
+
+  .bc-table tbody tr:nth-child(even) {
+    background-color: #eaeef2;
+  }
+
+  .bc-table tbody th code {
+    word-wrap: break-word;
+    width: 100%;
+    position: relative;
+    display: inline-block;
+  }
+
+  .bc-table .bc-platforms th {
+    width: auto;
+    padding: 5px 0 2px;
+    border-top: 0;
+    border-bottom: 2px solid #222;
+    border-left: 2px solid #222;
+    font-size: 28px;
+    font-size: 1.55556rem;
+  }
+
+  .bc-table .bc-platforms span {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute!important;
+    width: 1px;
+  }
+
+  .bc-table .bc-platforms .bc-platform-desktop:before {
+    background-image: url("https://developer.mozilla.org/static/platforms/desktop.d6def92f82da.svg");
```

```suggestion
    background-image: url("/bcd/platforms/desktop.svg");
```
And copy the SVGs from https://github.com/mozilla/kuma/tree/master/jinja2/includes/icons/platforms into `client/public/bcd` or something.